pageserver/src/tenant/timeline/timeline_wrappers.rs

```diff
@@ -0,0 +1,169 @@
+//! [`Timeline`] instances are rarely supposed to be used directly.
+//!
+//! Inside, such timelines hold the only instance [`tokio::sync::watch::Sender`],
+//! that gets dropped/changed based on certain events.
+//! There's a main `Arc<Timeline>` instance present in [`crate::tenant::Tenant`],
+//! which gets dropped on timeline deletion.
```

Comment:
I don't see any `tokio::sync::watch::Sender`s here

Reply:
That was referring to `[Timeline]`, reworded.